src/Relude/Extra/Validation.hs

```diff
@@ -143,6 +145,9 @@ Success 8
 
 >>> liftA2 (+) a c
 Failure ["Not correct"]
+
+>>> [x | Success x <- [ha <*> e, c <* e, d *> e, liftA2 (+) d e]]
```

Comment:
Heh, I wish these variable names could make a little sense, so the examples could become useful :cry: 
@chshersh should I create an issue for refactoring this?

Reply:
@vrom911 Yes, please!

Reply:
https://github.com/kowainik/relude/issues/221